reahl-web/reahl/web/fw.py

```diff
@@ -2786,20 +2788,22 @@ def define_static_files(self, path, files):
     def start(self, connect=True):
         """Starts the ReahlWSGIApplication by "connecting" to the database. What "connecting" means may differ
            depending on the persistence mechanism in use. It could include enhancing classes for persistence, etc."""
-        self.should_disconnect = connect
-        context = ExecutionContext(name='%s.start()' % self.__class__.__name__).install()
-        context.config = self.config
-        context.system_control = self.system_control
-        if connect:
-            self.system_control.connect()
-        
+        if not self.started:
```

Comment:
Its better to break if already started - because thats what youd expect when starting something twice.